change/monosize-190af040-06c8-4534-88ab-f97941ef976d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

we use zero based versioning - minor would meen breaking change

```suggestion
  "type": "patch",
```